src/arrayMethodSort.js

```diff
@@ -4,9 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else {
+        return -1;
+      }
+    },
+  ) {
+    let saveValue;
+
+    for (let j = 1; j < this.length; j++) {
+      for (let i = 1; i < this.length; i++) {
+        if (compareFunction(this[i - 1], this[i]) > 0) {
+          saveValue = this[i - 1];
+          this[i - 1] = this[i];
+          this[i] = saveValue;
+        }
+      }
+    }
```

Comment:
Consider optimizing the algorithm as bubble sort has O(n^2) complexity, which is not efficient for large arrays. While it's acceptable for this task, using more efficient algorithms like quicksort or mergesort would be better in production code.